org.freedesktop.Sdk.Extension.texlive.appdata.yaml

```diff
@@ -0,0 +1,16 @@
+---
```

Comment:
this file cannot be YAML. It must be XML.

Reply:
Well, what I mean more precisely is, yes it *can* be YAML per https://www.freedesktop.org/software/appstream/docs/sect-AppStream-YAML.html but I am genuinely unsurprised that support for that hasn't been tested.

Reply:
Yeah I don't think we want to promote another format for appdata on here. It should have been a separate PR anyway.